<?xml version="1.0"?>

<!--|
    Copyright (C) 2019 Learning Algorithms and Systems Laboratory, EPFL, Switzerland
    Authors:  Harshit Khurana
    email:    harshit.khurana@epfl.ch
    website:  lasa.epfl.ch

    This file is part of i_am_project EPFL (impact aware manipulation).
|-->

<launch>

  <node pkg = "i_am_project" type = "inertia_QP"  name = "inertia_QP" output="screen"/>

</launch>

